cheryl-cheng/SEPER
# Completed file upload and form submission

Summary

- Added a file uploader to the suggestion page that parses text and populates the form
- Updated existing front end code to work with the new way that these form values are handled

How to setup/test

- Make sure to do 'npm install' from the frontend directory
- From the root directory (SEPER), run "npm start" which should boot up both front and back end
- Copy paste the text below into a .txt file and upload it then submit form

title:Test-Driven Development of Automotive Software Functionality
authors:Yogesh. T., Vimala. P.
source:2020 Third International Conference on Smart Systems and Inventive Technology (ICSSIT) Smart Systems and Inventive Technology (ICSSIT)
pubyear:2020
volume:3
numpages:3
doi:10.1109/ICSSIT48917.2020.9214078
sepractice:TDD
